pkg/webui/components/file-input/index.js

```diff
@@ -31,6 +31,8 @@ const m = defineMessages({
   changeFile: 'Change file…',
   noFileSelected: 'No file selected',
   fileProvided: 'A file has been provided',
+  largeFileWarning:
+    'Providing files larger than 3MB can cause issues during the import process. We recommend you to split such files up into multiple smaller files and importing them one by one.',
```

Comment:
I realize now that the feature cannot be hardcoded like that into the component. Keep in mind that the component is just an input component. The file size warning is contextual to the device import feature. The `<FileInput />` component is not aware of any split-up requirements.

As such, both the warning threshold (10MB) and the warning message itself should be passed as props.